Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+Anyone can take the role of Proposer as long as they are knowledgeable in the task requirements and completion criteria. 
+
+**Assignee**
+^^^^^^^^^^^^
+The Assignee is the member of the Commissioning Science team expected to oversee the task during the preparation and execution phase. 
+They are expected to collaborate with the Proposer during task preparation, and are directly responsible for ensuring the task is Ready for execution at the summit. 
+They are expected to track task progress and advocate for task completion during the task planning meetings when necessary. 
+
+In addition, there is the role of planning the tasks for execution once they have been set to READY. 
+This is expected to be carried out in accordance with the priority assigned to the task by the Commissioning Science team. 
+While awaiting execution, tickets will remain in the READY state and their priority may be updated as needed.
+Ticket Assignees are expected to track the status of execution and advocate for planning of tasks as needed. 
+
+And finally, there is the role that is filled by the person responsible for task execution at the summit, in most cases an Observing Specialist. 
+Although responsible for execution of the task (arguably the most import step), this person does not have any defined responsibilities in the JIRA Workflow itself. 
+However, they do hold the final say of whether a task can be executed on the summit (as a final evaluation is implicit in their role), and have full discretion to transition tickets back to Preparation if they are found to be lacking instruction. 
+
+Overview of Workflow
+--------------------
+Here we provide a bulleted list illustrating an example workflow. 
+
+* Stakeholder requires an observing task to be completed at the summit, and describes the task in an LVV ticket, SITCOM ticket, or Confluence page. 
+* Stakeholder proposes a new BLOCK Jira ticket, providing links to supporting material and assuming responsibility of Proposer. 
+* New BLOCK Jira ticket is reviewed by Commissioning Science team during one of the regular, weekly planning meetings, and accepted/rejected for scheduling. 
+* New BLOCK Jira ticket is given a priority for scheduling and a Commissioning Scientist is Assigned the ticket, taking on the responsibility of Assignee. 
+* Proposer and Assignee work together to prepare BLOCK ticket for execution.
+* BLOCK ticket is marked as READY by Assignee, and added to the queue of tasks to be executed. 
+* BLOCK ticket is planned for execution at the summit by the night planner. 
+* BLOCK ticket is executed at the summit by Observing Specialist.
+* Data is analyzed by the Proposer and ticket is reported on supporting tickets (e.g. SITCOM). 
+* Proposer verifies observing task is completed and transitions ticket to Done to close issue.
+
```

Comment:
Before starting the new section, maybe would be a good idea to add a diagram focused on the responsibilities, rather than in the ticket states. If you like this idea, here is a suggestion:

https://mermaid.live/edit#pako:eNq9Vd9r2zAQ_leECqOFJKRrmyYODJLYhj2slCbrYPUetOgSi8iSkeS2adP_fZJ_xiUbfQgzftBJd9_pvrsPveKlpIA9vFYkjdHCH0cC2U-bLQd0q2QqNSi0Ypx7JxPfHwaDjjZKbsA7uQpHwaRfmt0nRk3sfU6fOzomKXjAOUs1HICjJVzYD4dhWMNN-7NLP_gbnIKlaWNBShQxTIoSLhgEg3BSw40urvv-9KNwE63ZWgAcqdIKbkH0RlflhrbgqxrTn173RxdtzHOHWWxQYsGVIlsPXXXsf_jad0Do9hj1-1LUtYeB_Wqcy6F_Ppt9FOereCSc0WNABRYp22_w8Qgs0lTT_XAa4WodReJ0bsgGYskpqLMIn_0qJVH11LlXa-c-k0nCrC0FE2s0XzIQhmmzH9rOh7rdL2g3U0AMaPSp2tdowZYbMLtaJ-NWHC3iytQU_d6iduoFkGS3L4zx-7w9B_BDKjuTT8zEzdRbjvfiDoHUrvklinN3ZTvgh3M28nQRLUE8RPjeTUl-7FkOQyas_dJStbb7pRfYQsWKrbPm5Hua7y9AGxSIR6akSCzvOsLv21VIsKCOczQ3kGp0D4qtWEFi5bgr1FSGF8rKw4JnWGbGOtNMOZ5vORGuATdsHdtuOemUQbmKCpbvAJr5vQGgQP_RWBfSzLvlx3W2l3c050dmglbasiXm9yrNcTXOTYd6_51w3MEJqIQwap-SV3efCJsYEoiwZ5cUViTjJsKReLOuJDNyvhVL7K0I19DBWQ7uM2JfoaTeTYn4KWVjA2VGqm_Fe5U_W29_AL6LUQ0

Reply:
Thanks for the suggestion. I like the checklist representation a little better than the diagram. The workflow is pretty linear, but the responsibilities go back and forth between roles depending on the state so that switching is tough to represent visually. I think I will skip adding the diagram here but keep the checklist.